reports/services.py

```diff
@@ -79,10 +79,15 @@ def resolve_place_divisions(place_json) -> Optional[list]:
         return None
     try:
         # When fetching the event, the division field is under location
-        if "location" in place_json:
-            place_json = place_json["location"]
-        divisions = place_json["divisions"]
-        return [d["ocd_id"] for d in divisions]
+        # NOTE: Some places, e.g helsinki:internet, might not have divisions
+        if (
+            place_json
+            and "location" in place_json
+            and "divisions" in place_json["location"]
+        ):
+            divisions = place_json["location"]["divisions"]
+            return [d["ocd_id"] for d in divisions]
+        return None
     except KeyError:
         return None
```

Comment:
The `if place_json` in the try clause is redundant with the `if not place_json` clause before the try clause, only one of them is needed.

One quite idiomatic Python (i.e. ["Easier to ask for forgiveness than permission"](https://docs.python.org/3/glossary.html#term-EAFP)) refactoring of the function:
```python
def resolve_place_divisions(place_json) -> Optional[list]:
    # When fetching the event, the division field is under location
    # NOTE: Some places, e.g helsinki:internet, might not have divisions
    try:
        return [d["ocd_id"] for d in place_json["location"]["divisions"]]
    except (IndexError, KeyError, TypeError):
        return None
```
Although I gotta say coming from a C++ background that looks a bit dangerous to me but AFAIK that's the more idiomatic Python way to do it. What's your take on this?